src/main/resources/swagger/api-docs.yaml

```diff
@@ -1183,6 +1183,54 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorReport'
+  /api/azure/v1/petManagedIdentity/{userEmail}:
+    post:
+      tags:
+        - Azure
+      summary: administratively gets or creates a pet managed identity for the specified user
```

Comment:
maybe some comment about "who" has permission to create PET managed identity on behalf of others?

Reply:
👍 .. will update this text.